kyo-prelude/shared/src/main/scala/kyo/Batch.scala

```diff
@@ -110,8 +110,6 @@ object Batch:
 
         case class Cont(op: Op[?, S], cont: Any => (Cont | A) < (Batch[S] & S & S2))
         def runCont(v: (Cont | A) < (Batch[S] & S & S2)): (Cont | A) < (S & S2) =
-            // TODO workaround, Flat macro isn't inferring correctly with nested classes
-            import Flat.unsafe.bypass
```

Comment:
I thought it was something related to nested classes/methods but it was due to the `Macro` trying to infer for `Cont | A` but then not summoning the existing `Flat[A]` in scope.